SPCTR-3218: Fix: Social Share block has extra margin left and margin right when gap between icons is set. (#3390)
brainstormforce/ultimate-addons-for-gutenberg: README.md

### x.x.x ###
* Fix: Post Carousel - Resolved an issue wherein the carousel dots would not appear centered.
* Fix: Testimonials - Resolved an issue wherein the Testimonial dots would not appear centered.

> * Fix: Social Share - Resolved an issue wherein the gap between icons would add unwanted margins to the sides of the wrapper.

Reviewer: Keep this entry above Testimonials, alphabetically.

Author: Done.

* Fix: Forms - Resolved an issue wherein the submit button would not take the padding set in the customizer.
* Fix: Call To Action - Resolved an issue wherein the line-height of the title would not appear responsive in the frontend.
* Fix: Inline Notice - Updated inconsistent default padding and removed unwanted top-margin in the editor and the frontend.

> * Fix: Social Share - Resolved an issue wherein the gap between icons is causing unwanted margin-left and margin-right to the wrapper.

Reviewer: ```suggestion
* Fix: Social Share - Resolved an issue wherein the gap between icons would add unwanted margins to the sides of the wrapper.
```

Author: Done.
